<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text recorder</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>361</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="QLabel" name="not_recorded_texts_label">
      <property name="text">
       <string>Need to record</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QScrollBar" name="recorded_texts_scroll_bar">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QListWidget" name="not_recorded_texts"/>
    </item>
    <item row="2" column="5">
     <widget class="QScrollBar" name="not_recorded_texts_scroll_bar">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="recorded_texts_label">
      <property name="text">
       <string>Recorded</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QListWidget" name="recorded_texts"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>43</y>
     <width>311</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Select device:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="devices"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Text:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_to_read">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Text to read will show up here...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="toggle_record">
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="save_to">
      <property name="text">
       <string>Save to...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
